synced_bcf_reader.c

```diff
@@ -708,6 +708,11 @@ static int _reader_fill_buffer(bcf_srs_t *files, bcf_sr_t *reader)
             ret = bcf_itr_next(reader->file, reader->itr, reader->buffer[reader->nbuffer+1]);
             if ( ret < -1 ) files->errnum = bcf_read_error;
             if ( ret < 0 ) break; // no more lines or an error
+            //set phasing of 1st value as in vcf v44
+            if ((ret = update44phasing(reader->header, reader->buffer[reader->nbuffer+1]))) {
```

Comment:
I am curious how this affects speed?

Reply:
Initially it was observed to make ~15% overhead. On recheck, last update had more overhead and tweaked further. It shows ~13% overhead with bcftools view.